README.md

```diff
@@ -4,7 +4,12 @@ This repository, developed in collaboration between Antmicro and Google's TF Lit
 
 ### Repo structure
 
-* `.github/workflows` - GH actions files (currently only generating `ipynb` files from `py` sources)
-* `notebooks` - interactive Google Colab demos
-  * `person detection` - person detection demo on nrf52840 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/antmicro/tensorflow-arduino-examples/blob/master/notebooks/person_detection/person_detection.ipynb)
+* `.github/workflows` - GH actions files
+  * `generate_ipynb_files.yml` - generating `ipynb` files from `py` sources
+  * `test_examples.yml` - building and testing examples
+* `examples` - TensorFlow Lite - Arduino example demos

```

Comment:
`examples` - scripts, tests and colab files for specific TensorFlow Lite examples